### PR TITLE
pymssql: init at 2.1.4

### PR DESCRIPTION
**Pull request by mredaelli**
###### Motivation for this change

Missing package from python collection

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg build python2.pkgs.pymssql python3.pkgs.pymssql

File: pkgs/development/python-modules/pymssql/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  version = "2.1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1yvs3azd8dkf40lybr9wvswvf4hbxn5ys9ypansmbbb328dyn09j";
+  };
+
+  buildInputs = [cython setuptools-git];
+  propagatedBuildInputs = [freetds];
+  doCheck = false;
```
Comment (FRidh) [inline pkgs/development/python-modules/pymssql/default.nix]: Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/pymssql/default.nix (hunk 1 above)
Comment (mredaelli) [reply pkgs/development/python-modules/pymssql/default.nix]: The tests require an SQL Server instance, so I just skipped them.
If that's OK, I'll add the comment right away

Comment (mredaelli) [reply pkgs/development/python-modules/pymssql/default.nix]: @FRidh I added the comment. Let me know if that's all

**Comment by FRidh** [comment]
@GrahamcOfBorg build python2.pkgs.pymssql python3.pkgs.pymssql

**Pull request merged**